sources/gov/kaunas/municipality/educational_institutions.yaml

```diff
@@ -0,0 +1,30 @@
+id: "gov/kaunas/municipality/educational_institutions"
+title: "Educational institutions"
+description: "CSV file containing data about Kaunas educational institutions"
+type: "source"
+source:
+  - "https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
+  - "csv:"
+provider: "gov/kaunas/municipality"
+objects:
+  educational_institution:
+    source: https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv
+    properties:
+      name:
+        type: "string"
+        source: "pavadinimas"
+      id:
+        type: "int"
+        source: "kodas"
+      address:
+        type: "string"
+        source: "adresas"
+      management_email:
+        type: "string"
+        source: "vad_email"
+      phone_number:
+        type: "string"
+        source: "vad_tel"
+      home_page:
+        type: "string"
+        source: "web"
```

Comment:
All names used here, like `educational_institution`, `name`, `id`, ... should also be defined in `vocabulary/` folder. Once you rename this file from `.yaml` to `.yml` you will get errors about that.

Vocabularies are used to normalize different naming between different data sources. Also this helps to provide 5 star data.